modules/src/ping.py

```diff
@@ -1,14 +1,15 @@
 import requests
 
 from templates.text import TextTemplate
+from urlparse       import urlparse
 
 def process(input, entities):
     output = {}
     try:
         url = entities['url'][0]['value']
-        strip_list = ['http://', 'https://', '/']
-        for item in strip_list:
-            url = url.strip(item)
+        if '//' not in url:
```

Comment:
This does not look elegant. Can be made better by trying to get the scheme of the parsed url, and adding it if does not exist; then parse it again.